server/controllers/questionController.js

```diff
@@ -0,0 +1,72 @@
+const questionModel = require('../models/questionModel')
+
+
+module.exports = {
+
+
+    addQuestion: function(req, res) {
+        const { name, option1, option2, option3, option4, correctOption, sectionId } = req.body
```

Comment:
can we add this code inside if(req.body)?

Reply:
added code in the if(req.body)